pull_request_template.md

```diff
@@ -0,0 +1,68 @@
+> problème (« issue ») GitHub ou la fiche Trello, le cas échéant.
+
+# Test instructions | Instructions pour tester la modification
+
+> Sequential steps (1., 2., 3., ...) that describe how to test this change. This
+> will help a developer test things out without too much detective work. Also,
+> include any environmental setup steps that aren't in the normal README steps
+> and/or any time-based elements that this requires.
+
+---
+
+> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de tester la
+> modification. Elles aideront les développeurs à faire des tests sans avoir à
+> jouer au détective. Veuillez aussi inclure toutes les étapes de configuration
+> de l’environnement qui ne font pas partie des étapes normales dans le fichier
+> README et tout élément temporel requis.
+
+# Release Instructions | Instructions pour le déploiement
+
+None.
+
+> Necessary steps to perform before and after the deployment of these changes.
+> For example, emptying the cache on a feature that changes the cache data
+> structure in Redis could be mentioned.
+
+---
+
+> Étapes nécessaires à exécuter avant et après le déploiement des changements
+> introduits par cette proposition. Par exemple, vider la cache suite à des
+> changements modifiant une structure de données de la cache pourrait être
+> mentionné.
+
+# Reviewer checklist | Liste de vérification du réviseur
+
+This is a suggested checklist of questions reviewers might ask during their
+review | Voici une suggestion de liste de vérification comprenant des questions
+que les réviseurs pourraient poser pendant leur examen :
+
+
+- [ ] Is it accessible? | Est-ce que c’est accessible?
+- [ ] Is it translated between both offical languages? | Est-ce dans les deux
+      langues officielles?
+- [ ] Is the code maintainable? | Est-ce que le code peut être maintenu?
+- [ ] Have you tested it? | L’avez-vous testé?
+- [ ] Are there automated tests? | Y a-t-il des tests automatisés?
+- [ ] Does this cause automated test coverage to drop? | Est-ce que ça entraîne
+      une baisse de la quantité de code couvert par les tests automatisés?
+- [ ] Does this break existing functionality? | Est-ce que ça brise une
+      fonctionnalité existante?
+- [ ] Does this change the privacy policy? | Est-ce que ça entraîne une
+      modification de la politique de confidentialité?
+- [ ] Does this introduce any security concerns? | Est-ce que ça introduit des
+      préoccupations liées à la sécurité?
+- [ ] Does this significantly alter performance? | Est-ce que ça modifie de
+      façon importante la performance?
+- [ ] What is the risk level of using added dependencies? | Quel est le degré de
+      risque d’utiliser des dépendances ajoutées?
+- [ ] Should any documentation be updated as a result of this? (i.e. README
+      setup, etc.) | Faudra-t-il mettre à jour la documentation à la suite de ce
+      changement (fichier README, etc.)?

```

Comment:
Should we cut down this reviewer section even more? If no one ever goes through and checks off boxes, it's a bit useless. For example, for api we can probably cut out a11y and translations. Or, for translations, something like "are you changing a string that needs to be translated in admin?"

I guess, I feel that most of these things should be thought of (and probably checked off or commented on) by the author. If this is basically the CDS template then maybe this discussion has already been had and settled.

Reply:
Yeah you are right, we could do a more targeted pull request template depending on the code base / repository. I'll remove these two items that you mentioned. 

Regarding the CDS standard, it's good, but not good enough if we don't make use of it. We might care for it a bit more if we build it ourselves.. and we might realize later that we'll need to include back some items that was initially there. Let's walk the path and see where it goes.